ocaml/sdk-gen/component-test/README.md

```diff
@@ -0,0 +1,20 @@
+## How Component Testing Works
+
+#### jsonrpc-server
+
+jsonrpc-server is a HTTP server designed to mock XAPI. This server executes the method specified in the request and returns a response containing the result. It is important to note that the server reads test data from JSON files in the spec/ directory. Each JSON file should contain a JSON object with the fields of test_id, method, params, and expect_result.
+
+#### jsonrpc-client
+
+jsonrpc-client is a client that imports the SDK and runs the functions, following these important details:
```

Comment:
It's also worth to mention that the test results/reports need to be collected from the client side. This is not in the client example.
An alternative is to design a simple framework for all kinds of SDKs.

Reply:
Update README for results/reports collection